frontend/src/metabase/collections/components/CreateCollectionForm/tests/setup.tsx

```diff
@@ -0,0 +1,54 @@
+/* istanbul ignore file */
+import { TokenFeatures, User } from "metabase-types/api";
+import {
+  createMockCollection,
+  createMockTokenFeatures,
+  createMockUser,
+} from "metabase-types/api/mocks";
+import { createMockState } from "metabase-types/store/mocks";
+import { setupEnterprisePlugins } from "__support__/enterprise";
+import { setupCollectionsEndpoints } from "__support__/server-mocks";
+import { mockSettings } from "__support__/settings";
+import { createMockEntitiesState } from "__support__/store";
+import { renderWithProviders } from "__support__/ui";
+import CreateCollectionForm from "../CreateCollectionForm";
+
+const ROOT_COLLECTION = createMockCollection({
```

Comment:
we also have ROOT_COLLECTION in `frontend/src/metabase/entities/collections/constants.ts` , can we reuse it somehow?